### ISSUE TITLE
No sound

### ISSUE THREAD

**Issue by GrahamDLL** [SamDel/ChromeCast-Desktop-Audio-Streamer#48]
Long story short ...

I have done a factory reset on all devices.  I am not able to get a Group to play music.

The Group works fine when I Cast from within the Chrome browser on a Windows computer.

Plus, I created and deleted groups in the Home app to try and get a result.  How do i make the deleted groups disappear from Streamer.  Thanks.
[log01.txt](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/3364611/log01.txt)

**Comment by GrahamDLL** [comment]
Same issue ... attempting to use one device and not use a group.  Thanks.

[log02.txt](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/3364658/log02.txt)

**Comment by SamDel** [comment]
It looks like the device can't load the stream from the application.
Can you check the firewall? Maybe the IP addresses changed because of the reset and the firewall needs to be updated.

When you do a 'Reset Settings' on the options tab the list of known devices is also cleared.
Or edit `C:\Users\<username>\AppData\Local\ChromeCast\...\<version>\user.config` manually.

**Comment by GrahamDLL** [comment]
Doh!  I installed a new router (that is also the DHCP server) a few days ago and everyone got new IP addresses.  Switching off the firewall fixed Streamer and music played.   I'll look for a more elegant solution tomorrow.  Thanks for your help.

**Comment by SamDel** [comment]
😊

**Comment by GrahamDLL** [comment]
The "proper" fix was to change the network connection from Public to Private.

I no idea why it went from "working" to "not working" (and I dpn't care since it's now fixed).

I fixed the "not disappearing" groups by uninstalling and deleting the user.config file and re-installing.  I haven't attempted a diagnosis because I probably created the problem by fiddling about trying to fix the "no sound" problem.

Thanks for your help.

**Comment by FA-Bubba** [comment]
Here is an issue I encountered that is not related to the Streamer App, but I'm describing here in case it may happen to anyone else:...

I have 8 pairs of multimedia stereo speakers that I stream to with the Streamer App via Groups, which keeps all of the speakers in sync.  At one point, I noticed one pair of speakers had dropped out of sync, and upon closer analysis, it was clear that this speaker set was receiving two independent streams, with different timing...

Coincidentally, this speaker pair had previously been synced to my phone via Bluetooth, but I had since told my phone to "forget" that speaker set, and they were not showing up in my phone's Bluetooth device list.  No other speaker pair had ever been synced to anything via Bluetooth...  

It turns out that this 'rogue' speaker pair had autonomously paired via Bluetooth with the PC that was streaming the audio content -- I had never paired any Bluetooth devices to this PC, so this Bluetooth connection was 'spontaneous'.  I went into the PC's Bluetooth control panel, and confirmed it was paired to the 'rogue' speaker pair, and that is why those speakers were playing two streams, with one out of sync.

I removed the Speakers from the PC's Bluetooth device list, and the problem was solved.  Now that I am aware, that will be the first diagnostic I consider if I ever hear an out-of-sync speaker.

Now that I have music in all rooms, I removed my Google Home Minis from every Group that they were previously assigned to, so now, I can independently set the volume for the Google Home devices, and all speaker pairs that my PC is streaming to...  This enables me to ask Google to do things, even if the music volume is low.
